.github/workflows/peerpod-ctrl_image.yaml

```diff
@@ -43,6 +43,7 @@ jobs:
             quay.io/confidential-containers/peerpod-ctrl:${{ github.sha }}
           push: true
           context: peerpod-ctrl
+          platforms: linux/amd64, linux/s390x
```

Comment:
I wonder if we should have add `,linux/ppc64le` as well for Power?

Reply:
I'd like to add `linux/ppc64le`, did not add it because I'm not sure why there is no `power` provider added in the Makefile https://github.com/confidential-containers/cloud-api-adaptor/blob/main/Makefile#L25

Reply:
Added `linux/ppc64le`.